configure.scss

```diff
@@ -1,9 +1,11 @@
-$_n-ui-variant: null !default;
+////
+/// @group n-ui features
+////
 
-$_n-ui-output-critical: false !default;
-
-// default feature values
-$_n-ui-features: (
+/// Configurable features
+/// @type List
+/// @private
+$n-ui-features: (
```

Comment:
Partly because they're public anyway, partly because some syntax highlighters (I.E. mine) don't like it 😬 - other than fuzzying up the diff it shouldn't make any difference.